salt/master.py

```diff
@@ -905,13 +905,10 @@ def __bind(self):
         )
 
         req_channels = []
-        tcp_only = True
         for transport, opts in iter_transport_opts(self.opts):
             chan = salt.transport.server.ReqServerChannel.factory(opts)
             chan.pre_fork(self.process_manager)
             req_channels.append(chan)
-            if transport != "tcp":
-                tcp_only = False
```

Comment:
**question** I see this removed - does it come into play/moved elsewhere or was it just unused?

Reply:
It looked to me like it was unused. IntelliJ/pycharm IDE highlighted it as such.